### DISCUSSION TITLE
AX3600砖了，boos大佬能帮下吗

### DISCUSSION THREAD

**Discussion by oppaioppai** [Boos4721/openwrt#146] [Q&A/答疑解惑] [answered]
本来没扩容好好的，想刷boos大佬的固件就按教程扩容，不知道哪步出错砖了，目前开机亮蓝灯，其他没反应，获取不了ip，看到恩山帖子，大佬回贴别人能帮忙救下就上来问问

**Comment by Boos4721** [comment, marked as answer]
https://www.right.com.cn/forum/thread-5635956-1-1.html OR DM [ME](https://t.me/boos4721)
